tests/cpp/helpers.cu

```diff
@@ -40,4 +47,24 @@ std::shared_ptr<DMatrix> RandomDataGenerator::GenerateDeviceDMatrix(bool with_la
       0, bins_);
   return m;
 }
+
+#if defined(XGBOOST_USE_RMM) && XGBOOST_USE_RMM == 1
```

Comment:
RMM code does not require compilation with NVCC, so I suggest moving the changes in this file to a .cpp/.cc file (helpers.cc) -- it will compile faster (NVCC is slow).

Reply:
When I tried including the RMM header in a .cc file, I got an error `cuda_runtime.h could not be located`.

Reply:
Never mind, I think I figured it out. I will be moving `SetupRMMResource()` to helper.cc.